spawner/run.py

```diff
@@ -32,9 +28,10 @@ def main():
     sys.stdout.flush()
     while True:
         try:
-            msg = bus_service.receive_queue_message(queueConf['queue_name'], peek_lock=False)
-            if msg.body is not None:
-                work = msg.body.decode("utf-8")
+            receiver = servicebus_client.get_queue_receiver(queueConf['queue_name'], receive_mode='peeklock')
```

Comment:
the old code is using `peek_lock=False` which means the message will be deleted from the service once it is sent.
To achieve the same effect in v7, you need to set the `receive_mode` to `ServiceBusReceiveMode.RECEIVE_AND_DELETE `.
```suggestion
            receiver = servicebus_client.get_queue_receiver(queueConf['queue_name'], receive_mode='peeklock')
```

Reply:
I have updated by your comments.